test/test.ts

```diff
@@ -21,40 +20,32 @@ test('return object', t => {
 		flags: {
 			unicorn: {shortFlag: 'u'},
 			meow: {default: 'dog'},
-			'--': true,
+			'--': true as unknown as {type: 'string'; isMultiple: true},
+			// TODO: can not model due to microsoft/TypeScript#17867
+			// A potential workaround is to use a symbol for '--' -> {flags: [argvDashDash]: true}
```

Comment:
One of the "rest index type" issues

Reply:
Use a proper URL over `microsoft/TypeScript#17867`